suite-native/module-settings/src/reducers/appSettingsReducer.ts

```diff
@@ -0,0 +1,28 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { ThemeColorVariant } from '@trezor/theme';
+import { RootState } from '@suite-native/state';
+
+export type AppColorScheme = ThemeColorVariant | 'system';
+
+export interface AppSettingsState {
+    colorScheme: AppColorScheme;
+}
+
+const initialState: AppSettingsState = {
+    colorScheme: 'system',
+};
+
+export const appSettingsSlice = createSlice({
+    name: 'appSettings',
+    initialState,
+    reducers: {
+        setColorScheme: (state, action: PayloadAction<AppColorScheme>) => {
+            state.colorScheme = action.payload;
+        },
+    },
+});
+
+export const selectThemeVariant = (state: RootState) => state.appSettings.colorScheme;
```

Comment:
```suggestion
export const selectedThemeVariant = (state: RootState) => state.appSettings.colorScheme;
```

Reply:
Why select**ed**? It doesn't makes sense because it's function that is going to select something, it's not actual value. Check also reselect docs https://github.com/reduxjs/reselect which is also using `select`

Reply:
Yeah, that doesn't make sense at all. What I would like best is `getSelectedThemeVariant` (viz this get prefix suggestion https://tppr.me/mwSfl)

Reply:
What's wrong with `select` prefix? :D Using `select` as prefix totally makes sense, it official recommended naming from reselect docs, it's explicitelly tell you that it's selector because it is usually not used anywhere else, which you can't say about `get` which is used pretty often not only in redux selector. 

Just keep it simple :D

Reply:
I am for `select`.